tests/e2e/Codeception_Basic/tests/_support/_generated/FunctionalTesterActions.php

```diff
@@ -12,5 +12,5 @@ trait FunctionalTesterActions
      */
     abstract protected function getScenario();
 
-    
+
```

Comment:
I would get rid of that line, there is two blank lines after `getScenario();` right now. I'm surprised this is not being caught by the CS fixer

Reply:
Its part of the e2e fixtures, which are ignored by it ( i think)